docker-compose.yml

```diff
@@ -4,7 +4,7 @@ services:
   web:
     build:
       context: ./app
-      dockerfile: Dockerfile.dev
```

Comment:
So docker-compose.yml is just the default docker-compose, so I think we should drop this. Dockerfile.prod only gets run on the server.

Reply:
change back to dev